internal/kafka/internal/presenters/kafka.go

```diff
@@ -52,9 +53,17 @@ func PresentKafkaRequest(kafkaRequest *dbapi.KafkaRequest) public.KafkaRequest {
 		InstanceType:            kafkaRequest.InstanceType,
 		ReauthenticationEnabled: kafkaRequest.ReauthenticationEnabled,
 		KafkaStorageSize:        kafkaRequest.KafkaStorageSize,
+		BrowserUrl:              fmt.Sprintf("%sapplication-services/streams/kafkas/%s/dashboard", validateBrowserUrl(browserUrl), reference.Id),
```

Comment:
Wdyt of adding the `/` after the `%s`? It may make it easier to read. 
```suggestion
		BrowserUrl:              fmt.Sprintf("%s/application-services/streams/kafkas/%s/dashboard", validateBrowserUrl(browserUrl), reference.Id),
```

In this case, you don't need to check if it has "/" at the end. It can be removed instead, if the browser url was defined with it. 

There's a function called [strings.TrimSuffix()](https://pkg.go.dev/strings#TrimPrefix) that you can use for this.

Reply:
I agree it would make it easier to read. :)

Reply:
@JameelB Could I just remove it from the default values and add it here then remove the validation on it altogether ?